apps/api-journeys/prisma/factories/nua9.ts

```diff
@@ -0,0 +1,1527 @@
+    }
+  })
+
+  // Third part of the prayer
+  const stepPrayer3 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 5
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: prayerCard2.id,
+      extraAttrs: {
+        label: 'Next',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
+        },
+        action: {
+          gtmEventName: 'click',
+          blockId: stepPrayer3.id
+        }
+      },
+      parentOrder: 3
+    }
+  })
+  const prayerImageId3 = uuidv4()
+  const prayerCard3 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'CardBlock',
+      parentBlockId: stepPrayer3.id,
+      extraAttrs: {
+        themeMode: ThemeMode.dark,
+        themeName: ThemeName.base,
+        coverBlockId: prayerImageId3
+      },
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      id: prayerImageId3,
+      journeyId: journey.id,
+      blockType: 'ImageBlock',
+      parentBlockId: prayerCard3.id,
+      extraAttrs: {
+        src: 'https://images.unsplash.com/photo-1519373344801-14c1f9539c9c?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1170&q=80',
+        alt: 'Decision',
+        width: 1920,
+        height: 1080,
+        blurhash: 'LqJs65}=R%so$,s:R*jb58Iqs:bH'
+      },
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: prayerCard3.id,
+      extraAttrs: {
+        content: '3/3',
+        variant: 'h6',
+        color: 'primary',
+        align: 'right'
+      },
+      parentOrder: 1
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: prayerCard3.id,
+      extraAttrs: {
+        content:
+          "I want to join life's adventure with you and become the person you made me to be.",
+        variant: 'subtitle1',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 1
+    }
+  })
+
+  // final part of the prayer
+  const stepPrayer4 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 6
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: prayerCard3.id,
+      extraAttrs: {
+        label: 'Amen',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
```

Comment:
```suggestion
          name: 'BeenhereRound'
```

Need to add to Icons list